frontend/src/metabase/dashboard/components/Dashboard/Dashboard.jsx

```diff
@@ -83,51 +83,39 @@ class DashboardInner extends Component {
     this.props.cancelFetchDashboardCardData();
   }
 
-  async loadDashboard(dashboardId) {
-    const {
-      editingOnLoad,
-      addCardOnLoad,
-      addCardToDashboard,
-      fetchDashboard,
-      initialize,
-      loadDashboardParams,
-      location,
-      setErrorPage,
-      isNavigatingBackToDashboard,
-    } = this.props;
-
-    initialize({ clearCache: !isNavigatingBackToDashboard });
-
-    loadDashboardParams();
-
-    const result = await fetchDashboard({
-      dashId: dashboardId,
-      queryParams: location.query,
+  async loadDashboard() {
+    const p = this.props;
```

Comment:
We don't use this convention anywhere. We either destructure props or use them as it is. I'd recommend destructuring

Reply:
Reverted back.